engine/frontend/src/App.js

```diff
@@ -1,23 +1,14 @@
 import './App.css';
-import Home from './components/Home';
-import Services from "./components/Services"
-import ServiceInfo from "./components/ServiceInfo"
-import CreateEnclave from './components/CreateEnclave';
-import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Home from './component/Home';
```

Comment:
Are there any tests for any of this? If not that's alright. Can you link some screenshots from this build in the comments?

Reply:
yepp; can do that. no tests atm.. will keep the commented code ( as will uncommenting it upcoming PRs)